tests/sessions/python/activity/no_activity.txtar

```diff
@@ -0,0 +1,19 @@
+start
+end
+
+-- main.py:main --
+import autokitteh
+from random import randint, randrange
```

Comment:
i need to see a test with a method, not just static functions

Reply:
Done